packages/fx-core/src/plugins/resource/bot/dotnet/plugin.ts

```diff
@@ -51,6 +52,22 @@ export class DotnetBotImpl extends TeamsBotImpl {
     return ResultFactory.Success(result);
   }
 
+  public async postLocalDebug(context: PluginContext): Promise<FxResult> {
+    await super.postLocalDebug(context);
+    const appSettingsPath = path.join(context.root, PathInfo.appSettingDevelopment);
+    try {
+      let appSettings = await fs.readFile(appSettingsPath, "utf-8");
```

Comment:
Not quite understand the logic, why we replace a regular expression with a placeholder? Are they the same thing? @JerryYangKai